pygmt/src/pygmtlogo.py

```diff
@@ -0,0 +1,275 @@
+"""
+pygmtlogo - Create and plot the PyGMT logo.
+The design of the logo is kindly provided by `@sfrooti <https://github.com/sfrooti>`_.
+"""
+
+from pathlib import Path
+
+import pygmt
+
+
+def pygmtlogo(  # noqa: PLR0915
+    self,
+    blackwhite=False,
+    darkmode=True,
+    hexshape=False,
+    wordmark=True,
+    orientation="horizontal",  # "horizontal" | "vertical"
+    position="jRT+o0.1c+w5c",  # -> use position parameter of Figure.image
```

Comment:
Default to `None` so that we can call `Figure.pygmtlogo` without a basmeap.
```suggestion
    position=None,  # -> use position parameter of Figure.image
```